feature/bgp/bgp_isis_redistribution/otg_tests/bgp_isis_redistribution_test/bgp_isis_redistribution_test.go

```diff
@@ -459,16 +472,11 @@ func verifyNonMatchingCommunityTelemetry(t *testing.T, dut *ondatra.DUTDevice, a
 	if commSet == nil {
 		t.Errorf("Community set is nil, want non-nil")
 	}
-	if deviations.BgpCommunityMemberIsAString(dut) {
-		cm := nonMatchingCommunityVal
-		if commSetMember := commSet.GetCommunityMember(); len(commSetMember) == 0 || !containsValue(commSetMember, oc.RoutingPolicy_DefinedSets_BgpDefinedSets_CommunitySet_CommunityMember_Union(oc.UnionString(cm))) {
-			t.Errorf("Community set member: %v, want: %s", commSetMember, cm)
-		}
-	} else {
-		cm, _ := strconv.ParseInt(fmt.Sprintf("%04x%04x", dummyAS, 200), 16, 0)
-		if commSetMember := commSet.GetCommunityMember(); len(commSetMember) == 0 || !containsValue(commSetMember, oc.RoutingPolicy_DefinedSets_BgpDefinedSets_CommunitySet_CommunityMember_Union(oc.UnionUint32(cm))) {
-			t.Errorf("Community set member: %v, want: %d", commSetMember, cm)
-		}
+
```

Comment:
I see 'bgp_community_member_is_a_string: true' for cisco, Why was this deviation removed?